site/content/entry/ldn-is-here/index.md

```diff
@@ -0,0 +1,159 @@
+If you're using the yuzu installer, you'll automatically be updated to the latest build. <br>	
+If you're not using the yuzu installer, please download it from our <a href= https://yuzu-emu.org/downloads/>downloads</a> page. <br>
+We highly recommend using our installer to always stay up-to-date on both Mainline and Early-Access builds.
+</p></div></article>
+
+## Local Wireless (LDN) Multiplayer
+
+yuzu's Local Wireless multiplayer (LDN) differs a little bit when compared to the Nintendo Switch.
+The Switch console uses the `LDN` service to scan for other Switch consoles using Wi-Fi and connects all players.
+Here, one player would act as the `host` and all others would connect to the host.
+
+yuzu, on the other hand, uses virtual rooms to allow players to create and join game lobbies within the room.
+Since this emulates the Switch's ability to do local wireless multiplayer, it doesn’t rely on Nintendo’s servers and does not require a Nintendo Network ID. 
+While on a real Switch you’d be limited to the people in your immediate vicinity, yuzu boasts a complex server/client infrastructure that forwards a game’s wireless communication across the internet.
+
+In order to get together with other players, you’re going to have to join the same room with yuzu’s room browser. 
+Out of the box, we provide you with a bunch of public rooms hosted across the world by yuzu.
+Creating and joining rooms is extremely easy in yuzu and can be done in just a few clicks.
+
+{{< imgs "./room_browser.png" >}}
+
+If you’re a verified user, you can create a public room for people to join. 
+These public rooms can be seen by anyone using the room browser, but you can also put a password on them to restrict users. 
+Remember to port forward, otherwise your friends won’t be able to connect!
+
+Unverified users aren’t left without options, though - they still have the ability to create unlisted rooms, direct connecting, and can join any yuzu hosted room.
+
+Do note that verified users will have their privileges revoked for violating any site policies while on the room chatroom. 
+Please respect the "recommended game" listed in publicly hosted rooms, as even unrelated games will add to the bandwidth load.
+
+{{< imgs "./room_chat.png" >}}
+
+**Note: You can use these rooms for LAN mode games too - instead of ZeroTier or Hamachi.**
+
+## Development
+
+FearlessTobi began the development by leveraging the existing virtual rooms infrastructure from [Citra](https://citra-emu.org).
+Due to code similarities between yuzu and Citra, this tried and tested infrastructure easily became the foundation of this feature.
+
+Now that the virtual rooms were taken care of, Tobi moved onto the `LDN` service and its protocols.
+Using the research and code from the [ldn_mitm](https://github.com/spacemeowx2/ldn_mitm) project, he implemented the necessary changes for yuzu instances to communicate with each other over these virtual rooms.
```

Comment:
```suggestion
Now that the virtual rooms were taken care of, Tobi moved onto the `LDN` service and its protocols.
Using the research and code from the [ldn_mitm](https://github.com/spacemeowx2/ldn_mitm) project, he implemented the necessary changes for yuzu instances to communicate with each other over these virtual rooms. And to prevent user IPs from being leaked when connected to rooms, Tobi modified the implementation to use fake IPs for routing.
```